starter-code/src/app/movies-data.ts

```diff
@@ -0,0 +1,122 @@
+export class Movie {
+  constructor(
```

Comment:
By the way, an interface could also work here, since we are not really using the constructor function, but creating literal objects.